hermes/bin/tests/integration/components/sleep_component/src/stub.rs

```diff
@@ -0,0 +1,83 @@
+use crate::{
```

Comment:
Have a look at what @rafal-ch is working on, we should not have any unused events inside any wasm module.